applications/plugins/image/initImage.cpp

```diff
@@ -21,6 +21,7 @@
 ******************************************************************************/
 #include <image/config.h>
 #include <sofa/helper/system/config.h>
+#include <sofa/helper/system/PluginManager.h>
```

Comment:
Maybe add `using sofa::helper::system::PluginManager;` here to avoid the long namespaces below :wink:

Reply:
you are picky, for one line only, it is nearly more work to add the using :wink: